app/file.go

```diff
@@ -621,6 +621,30 @@ func (a *App) GetFileInfo(fileId string) (*model.FileInfo, *model.AppError) {
 	return result.Data.(*model.FileInfo), nil
 }
 
+func (a *App) GetFileThumbnail(fileId string) ([]byte, *model.AppError) {
+
+	fileInfo, err := a.GetFileInfo(fileId)
+	if err != nil {
+		mlog.Error(
+			fmt.Sprintf("File not found when getting file thumbnails, err=%v", err),
```

Comment:
I'm inclined to think we should just return the error here (and below) vs. logging it: it's hard to predict how plugins might use this API, but logging errors that the plugin chooses to handle might just result in spamming the system log.